kani-driver/src/args.rs

```diff
@@ -179,6 +179,16 @@ pub struct KaniArgs {
     /// Disable CBMC's slice formula which prevents values from being assigned to redundant variables in traces.
     #[structopt(long, hidden_short_help(true), requires("enable-unstable"))]
     pub no_slice_formula: bool,
+
+    /// Randomize the layout of structures. This option can help catching code that relies on
+    /// a specific layout chosen by the compiler that is not guaranteed to be stable in the future.
+    /// See the `-Zrandomize-layout` argument of the rust compiler.
+    #[structopt(long)]
+    pub randomize_layout: bool,
+
+    /// Specify the layout seed for the `--randomize-layout` option.
+    #[structopt(long, requires("randomize-layout"))]
+    pub layout_seed: Option<u64>,
```

Comment:
Can you join these arguments together?

Reply:
Do you mean make 1 argument out of them or just remove the space in between?

Reply:
Make 1 argument out of them.

Reply:
Not sure what type it should take then
`Option<Option<u64>>`?

Reply:
Or even how the parser would behave
`--randomize-layout <seed>`, with no seed for default?
but then if one writes `--randomize-layout --other-option` the other option will be parsed as the seed and probably fail.
Otherwise `--randomize-layout <default | seed>`, but the default version is a bit verbose

Reply:
I would expect the parser to be smart enough to correctly parse `--randomize-layout --other-option`.

I think you are correct that with today's implementation, we cannot omit `default` though. @sanjit-bhat bumped into a similar problem with the concrete playback. Maybe we should consider moving to clap completely.

Reply:
Indeed it should be `Option<Option<T>>` and you can find that documented here: https://docs.rs/structopt/latest/structopt/#type-magic

I'm not sure if it requires the `=`  or nor, but it's at least documented that way. (i.e. `--randomize-layout=seed`)

Reply:
Cool that also makes it nicer because that's the interface I used to pass a single argument to `format_unit_test` 
<img width="421" alt="image" src="https://user-images.githubusercontent.com/8451179/188726167-e608cc1c-c45f-4648-9e1e-5b86a95128b5.png">
So I'll be able to simply pass the config directly